packages/block-editor/src/components/inserter/style.scss

```diff
@@ -229,6 +220,7 @@ $block-inserter-search-height: 38px;
 	border: $border-width solid $light-gray-500;
 	border-radius: $radius-round-rectangle;
 	align-items: center;
+	min-height: 150px;
```

Comment:
Can we use a variable here? Or at least something grid friendly like `$grid-unit-60 * 3`? I know no-one will really notice this, but I just don't like arbitrarily picked numbers :)